### PR TITLE
add top level package json, gitignore and npm install

### PR DESCRIPTION
**Pull request by wibjorn**
- using node 15.9, npm 7.5.3
- add top level package.json for monorepo management
- add .gitignore
- npm install and commit lock files

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+	},
+	"repository": {
+		"type": "git",
+		"url": "https://github.com/microsoft/atlas-design"
+	},
+	"workspaces": {
+		"packages": [
+			"css"
+		]
+	},
```
Comment (jdanyow) [inline package.json:16]:
```json
"workspaces": ["css"],
```

File: package.json [hunk 2]
```diff
@@ -0,0 +1,26 @@
+{
+	"name": "@microsoft/atlas-design",
+	"version": "1.0.0",
+	"description": "The Atlas Design System used by Microsoft's Developer Relations platform.",
```
Comment (jdanyow) [inline package.json:4]:
```
"private": true
```
to ensure tooling never publishes the monorepo package
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#private

Nice to have homepage as well https://docs.npmjs.com/cli/v7/configuring-npm/package-json#homepage

File: package.json [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+	},
+	"keywords": [
+		"microsoft",
+		"atlas",
+		"design-system",
+		"css",
+		"scss"
+	],
+	"author": "Microsoft Corporation",
+	"license": "MIT"
```
Comment (jdanyow) [inline package.json:25]: Thinking this PR should include a `LICENCE` file. Alternatively, there's a method to add one via the UI:
https://docs.github.com/en/github/building-a-strong-community/adding-a-license-to-a-repository

**Review by jdanyow — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (wibjorn) [reply package.json:25]: Microsoft open source took care of that for us: https://github.com/microsoft/atlas-design/blob/main/LICENSE

File: package.json (hunk 1 above)
Comment (wibjorn) [reply package.json]: whoops, updated

File: package.json (hunk 2 above)
Comment (wibjorn) [reply package.json:4]: updated

**Review by jdanyow — APPROVED** [review]

**Pull request merged**